addons/test_mail/tests/test_ir_actions.py

```diff
@@ -180,6 +183,157 @@ def test_action_next_activity_from_x2m_user(self):
             }],
         )
 
+    def test_action_next_activity_formview(self):
+        email_activity_type = self.env.ref('mail.mail_activity_data_email')
+        email_activity_type.write({'default_note': '<p>Default note for email</p>'})
+        with Form(self.env['ir.actions.server'], view='base.view_server_action_form') as f:
+            self.assertEqual(f.name, '')
+            f.model_id = self.res_partner_model
+            f.state = 'next_activity'
+            self.assertTrue(f._get_modifier('activity_plan_id', 'invisible'))
+            self.assertEqual(f.name, 'Create Activity')
+            self.assertTrue(f._get_modifier('activity_type_id', 'required'))
+            f.activity_type_id = self.env.ref('mail.mail_activity_data_todo')
+            self.assertEqual(f.name, 'Create To-Do')
+            self.assertEqual(f.activity_summary, 'To-Do')
+            self.assertEqual(f.activity_date_deadline_range, 1)
+            self.assertEqual(f.activity_date_deadline_range_type, 'days')
+            self.assertEqual(f.activity_user_type, 'specific')
+            self.assertFalse(f.activity_user_id)
+            self.assertFalse(f.activity_note)
```

Comment:
If you picked "f" because lines got too long somehow; Consider `self.assertDict(f.read(fields), {'id': f.id, ...})`